packages/legacy-scripting-runner/index.js

```diff
@@ -694,22 +693,14 @@ const legacyScriptingRunner = (Zap, zcli, input) => {
         }
 
         if (options.defaultToResponse && !result) {
-          try {
-            result = zcli.JSON.parse(response.content);
-          } catch {
-            result = {};
-          }
```

Comment:
We could already use `response.json` here, but now `response.data`.

Reply:
Unfortunately, legacy-scripting-runner has to (or is better to) work with older versions to core. Its `package.json` [states](https://github.com/zapier/zapier-platform/blob/eff0e5c3991e6089d4afe911c1870c06d427da38/packages/legacy-scripting-runner/package.json#L40) that it should work as long as core >= 7.6.0. So can we undo the change here?